irods_testing_environment/irods_config.py

```diff
@@ -278,6 +278,86 @@ def set_hostnames(docker_client, docker_compose_container):
         raise RuntimeError('failed to configure host resolution on some service')
 
 
+def configure_hello_script(docker_client, compose_project):
+    """Configure hello script for iRODS tests.
+
+    Arguments:
+    docker_client -- docker client for interacting with the docker-compose project
+    compose_project -- compose.Project in which the iRODS servers are running
+    """
+    def modify_script(docker_client, docker_compose_container, script):
+        chown_msiexec = 'chown irods:irods {}'.format(os.path.dirname(script))
+        copy_from_template = 'cp {0}.template {0}'.format(script)
+        make_script_executable = 'chmod 544 {}'.format(script)
+
+        on_container = docker_client.containers.get(docker_compose_container.name)
+        if execute.execute_command(on_container, chown_msiexec) is not 0:
+            raise RuntimeError('failed to change ownership to msiExecCmd_bin [{}]'
+                               .format(on_container.name))
+
+        if execute.execute_command(on_container,
+                                   copy_from_template,
+                                   user='irods',
+                                   workdir=context.irods_home()) is not 0:
+            raise RuntimeError('failed to copy hello.template template file [{}]'
+                               .format(on_container.name))
+
+        if execute.execute_command(on_container,
+                                   make_script_executable,
+                                   user='irods',
+                                   workdir=context.irods_home()) is not 0:
+            raise RuntimeError('failed to change permissions on hello script [{}]'
```

Comment:
Revisit error messages (i.e. `failed to copy hello.template template file`).

Reply:
Log messages are correct and good enough.

We can always improve them more if needed.

Resolving.